docs/adding_new_strategy.md

```diff
@@ -144,7 +144,7 @@ export async function hydratePayloadDependencies(
 ```
 
 ---
-
-> ⚠️ **Notes**: Documentation and testing are not specified here because it is difficult to predict what is needed. However, you are also responsible for adding them.
+> [!NOTES] 
```

Comment:
```suggestion
> [!WARNING] 
```

Would see a warning here

Also note that it's `NOTE` instead of `NOTES` anyway ^^